HW-03/main.py

```diff
@@ -0,0 +1,33 @@
+text = 'I love Python '           
+text *= 42
+print(text)
+
+
+one_month_of_the_year = 10 / 12   # Для вирахування  значення одного місяця в році
+age = 23                         
+month_of_year = 12               
+remaining_months = 5             
+age_in_month = age * month_of_year +(remaining_months * one_month_of_the_year)
+print(age_in_month)
+
+age_in_years = age
+print(age_in_years)
+
+age_in_years = str(age)
+name = 'Volodymyr'
+my_age = 'My name is ' + (name) +' I`m ' + (age_in_years) + ' years old.'
+print(my_age)
+
+number = 1
+print (number > 2)
```

Comment:
There shouldn't be a space in a call of print function. Just `print(number > 2)`. Fix it here and below